### PR TITLE
Fix wikigeo polygon

### PR DESCRIPTION
**Pull request by tzufgoogle**
Fixed Manhattan polygon in wikidata to match the polygon in map_structure. 

Build, test, run working

### CODE REVIEW

File: cabby/data/wikidata/query_test.py [hunk 1]
```diff
@@ -14,6 +14,9 @@
 
 '''Tests for query.py'''
 
+import sys
+sys.path.append("/home/tzuf_google_com/dev/cabby")
+
```
Comment (jasonbaldridge) [inline cabby/data/wikidata/query_test.py:19]: Remove.

File: cabby/geo/util.py [hunk 2]
```diff
@@ -346,3 +346,16 @@ def get_linestring_distance(line: LineString) -> int:
   trans_line = transform(project, line)
 
   return round(trans_line.length)
+
+def point_str_to_sheply_point(point_str: Text) -> Point:
```
Comment (jasonbaldridge) [inline cabby/geo/util.py:350]: sheply -> shapely

**Review by jasonbaldridge — CHANGES_REQUESTED** [review]
Thanks! Just a couple of minor things.

File: cabby/data/wikidata/query_test.py (hunk 1 above)
Comment (tzufgoogle) [reply cabby/data/wikidata/query_test.py]: removed

File: cabby/geo/util.py (hunk 2 above)
Comment (tzufgoogle) [reply cabby/geo/util.py]: changed

**Review by tzufgoogle — COMMENTED** [review]
Made the changes.

**Review by jasonbaldridge — APPROVED** [review]
LGTM

**Pull request merged**